packages/@glimmerx/core/tests/modifier-tests.ts

```diff
@@ -7,7 +7,9 @@ import { renderComponent, didRender } from '..';
 module('Modifier Tests', () => {
   test('Supports the on modifier', async (assert) => {
     class MyComponent extends Component {
-      static template = hbs`<button {{on "click" this.incrementCounter}}>Count: {{this.count}}</button>`;
```

Comment:
This line hit the line length limit for the formatter, so it will wrap and indent.